bin/wott-agent.sh

```diff
@@ -0,0 +1 @@
+/usr/bin/wott-agent
```

Comment:
Shouldn't this be `/usr/local/bin/wott-agent`?

Reply:
No, it’s correct